style.css

```diff
@@ -3,8 +3,8 @@ Theme Name: CyberChimps
 Theme URI: http://cyberchimps.com/cyberchimps-free/
 Author: CyberChimps
 Author URI: http://www.cyberchimps.com/
-Description: A Free HTML5 Responsive Drag & Drop Starter WordPress Theme by CyberChimps WordPress Themes featuring Twitter Bootstrap, and responsive touch friendly theme options. CyberChimps Free has visible edit links in the customizer preview. For Support - <a href="https://cyberchimps.com/forum/free/cyberchimps/" target="_blank"> Support forum</a> (https://cyberchimps.com/forum/free/cyberchimps/).
-Version: 1.5
+Description: A Free Gutenberg Compatible, HTML5 Responsive Drag & Drop Starter WordPress Theme by CyberChimps WordPress Themes featuring Twitter Bootstrap, and responsive touch friendly theme options. CyberChimps Free has visible edit links in the customizer preview. For Support - <a href="https://cyberchimps.com/forum/free/cyberchimps/" target="_blank"> Support forum</a> (https://cyberchimps.com/forum/free/cyberchimps/).
```

Comment:
Add gutenberg compatible after 'CyberChimps Free has visible edit links in the customizer preview ' like
'CyberChimps Free has visible edit links in the customizer preview and is gutenberg compatible'